core/DataTable/Row.php

```diff
@@ -521,7 +521,13 @@ private function getColumnValuesMerged($operation, $thisColumnValue, $columnToSu
                 $newValue = null;
                 break;
             case 'max':
-                $newValue = max($thisColumnValue, $columnToSumValue);
+                if (!$thisColumnValue) {
+                    $newValue = $columnToSumValue;
+                } elseif (!$columnToSumValue) {
+                    $newValue = $thisColumnValue;
+                } else {
+                    $newValue = max($thisColumnValue, $columnToSumValue);
+                }
                 break;
             case 'min':
```

Comment:
The implementation of `min` looks like it is not providing the expected result for:

- `$thisColumnValue = 0`
- `$columnToSumValue = null`

Current result: `$newValue = null`

Expected result: `$newValue = 0`